azurerm/resource_arm_storage_queue_test.go

```diff
@@ -217,8 +245,62 @@ func testAccAzureRMStorageQueue_requiresImport(rInt int, rString string, locatio
 
 resource "azurerm_storage_queue" "import" {
   name                 = "${azurerm_storage_queue.test.name}"
-  resource_group_name  = "${azurerm_storage_queue.test.resource_group_name}"
   storage_account_name = "${azurerm_storage_queue.test.storage_account_name}"
 }
 `, template)
 }
+
+func testAccAzureRMStorageQueue_metaData(rInt int, rString string, location string) string {
+	template := testAccAzureRMStorageQueue_template(rInt, rString, location)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_storage_queue" "test" {
+  name                 = "mysamplequeue-%d"
```

Comment:
Should this be 
```suggestion
  name                 = "acctestQueue-%d"
```

Reply:
it doesn't need to be (and infact you can't use upper-case characters here) - since this won't be picked up by the Dalek since this is a Data Plane API rather than a Resource Manager API; instead this'll be picked up by the Storage Account being prefixed with `acctest` - so I think we're good here 👍